tests/mail/tests.py

```diff
@@ -804,7 +806,7 @@ def test_send_unicode(self):
         self.assertEqual(num_sent, 1)
         message = self.get_the_message()
         self.assertEqual(message["subject"], '=?utf-8?q?Ch=C3=A8re_maman?=')
-        self.assertEqual(message.get_payload(decode=True).decode(), 'Je t\'aime très fort')
+        self.assertEqual(message.get_payload(), 'Je t\'aime très fort')
```

Comment:
Some behavior has changed here. On the file-backend, the output of `get_payload(decode=True)` is now `b"Je t'aime tr\xe8s fort"` as opposed to `b"Je t'aime tr\xc3\xa8s fort"` originally (and with the other backends). I guess this might be a bug.